iw/filter_gui.py

```diff
@@ -902,6 +917,51 @@ def _active(info):
                          "\n".join(members), "", "", "")
                 _addDeviceData(device_data)
 
+            # handle imsm/isw separately without pyblock/dmraid
+            mduuids = []
+            for mdmember in mdraids:
+                # Put the container in mdadm.conf to be consistent with the way
+                # it is done in the storage code. Otherwise the names will vary
+                # depending on whether the user visited the filter the first
+                # time through. If the devicetree was already populated this
+                # should have no effect since mdadm.conf will already be
+                # written out. Note that the fwraid arrays themselves are
+                # not in mdadm.conf until after they are activated, resulting
+                # in them having names like md127.
+                uuid = udev_device_get_md_uuid(mdmember)
+                name = "md%d" % len(mduuids)
+                ensure_mdadm_conf_entry("/dev/" + name, uuid)
+                if uuid not in mduuids:
+                    mduuids.append(uuid)
```

Comment:
This would be slightly more optimal if ``mduuids`` was a ``set()``, but that's just a nitpick.